common/src/main/java/com/wynntils/screens/settings/WynntilsBookSettingsScreen.java

```diff
@@ -255,6 +259,7 @@ protected void doInit() {
 
     @Override
     public void doRender(GuiGraphics guiGraphics, int mouseX, int mouseY, float partialTick) {
+        this.renderBackground(guiGraphics, mouseX, mouseY, partialTick);
```

Comment:
Why is this change necessary?

Reply:
Otherwise, the modmenu background is still visible and creates some visual issues. Can't call the super render method either as that also causes issues so just need the background

Reply:
What if we override renderBackground so it only renders a background if "McUtils.mc().level" is null?

Reply:
https://github.com/Wynntils/Artemis/pull/2703/commits/5223418af691789fb205d36553bf12fe177f1d57 Like this?

Reply:
Yes